robolectric-sandbox/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -99,15 +99,8 @@ public boolean shouldAcquire(String name) {
       return true;
     }
 
-    // Internal android R class must be loaded from the framework resources in the framework jar.
-    if (name.matches("com\\.android\\.internal\\.R(\\$.*)?")) {
-      return true;
-    }
-
-    // Android SDK code almost universally refers to com.android.internal.R, except
-    // when refering to android.R.stylable, as in HorizontalScrollView. arghgh.
-    // See https://github.com/robolectric/robolectric/issues/521
-    if (name.startsWith("android.R")) {
+    // android.R and com.android.internal.R classes must be loaded from the framework jar
+    if (name.matches("(android|com\\.android\\.internal)\\.R(\\$.+)?")) {
```

Comment:
Yeah, I think so, although, you can never say never :-D I think this will go away when we check the jar as the source of truth for what to acquire though anyway, so I'm not fussed about this :-)